bucket/gnucash.json

```diff
@@ -2,17 +2,25 @@
     "homepage": "https://www.gnucash.org/",
     "description": "Personal and small-business financial-accounting software",
     "license": "GPL-2.0-or-later",
-    "version": "3.2",
-    "url": "https://downloads.sourceforge.net/project/gnucash/gnucash%20%28stable%29/3.2/gnucash-3.2.setup.exe#/gnucash-setup.exe",
-    "hash": "0e06acfe7a9746eacf5282ba27919625b51c3d51fd548e6a529f78ff759ed70f",
+    "version": "3.6",
+    "url": "https://github.com/Gnucash/gnucash/releases/download/3.6/gnucash-3.6.setup.exe",
+    "hash": "cb8ba5aa13dce6c9caedfb774abef47b8e352c2cda7d762d3f74840f1b12766a",
+    "bin": "bin/gnucash.exe",
     "innosetup": true,
     "shortcuts": [
         [
             "bin/gnucash.exe",
             "GnuCash"
         ]
     ],
+    "checkver": {
+        "github": "https://github.com/Gnucash/gnucash"
+    },
     "autoupdate": {
-        "url": "https://downloads.sourceforge.net/project/gnucash/gnucash%20%28stable%29/$version/gnucash-$version-setup.exe#/gnucash-setup.exe"
+        "url": "https://github.com/Gnucash/gnucash/releases/download/$version/gnucash-$version.setup.exe",
+        "hash": {
+            "url": "https://github.com/Gnucash/gnucash/releases",
+            "find": "([A-Fa-f\\d]{64}).*gnucash-$version.setup.exe"
```

Comment:
```suggestion
            "regex": "$sha256.*?$basename"
```

Reply:
I didn't know you could do that, thanks!